### ISSUE TITLE
cwlexec doesn't support inputs of type Array<enum>

### ISSUE THREAD

**Issue by mscheremetjew** [IBMSpectrumComputing/cwlexec#30]
cwlexec reports the following and exits:
The variable type of the field [type] is not valid, "a valid CWL type" is required.

if the input port is defined like that:
```
inputs:
  - id: applications
    type:
      type: array
      items:
        type: enum
        name: applications
        symbols:
          - PfamA
          - TIGRFAM
```
on the other hand cwltool & cwl-runner is accepting those type definitions.

**Comment by mscheremetjew** [comment]
here is a simple test case for reproduction:
[test.cwl.txt](https://github.com/IBMSpectrumComputing/cwlexec/files/2415001/test.cwl.txt)
[test.input.yaml.txt](https://github.com/IBMSpectrumComputing/cwlexec/files/2415002/test.input.yaml.txt)

**Comment by skeeey** [comment]
This should be a bug, we will work on it

**Comment by mscheremetjew** [comment]
@skeeey Thanks a lot for letting us know.

**Comment by skeeey** [comment]
Fixed at c5990c4
